src/translations/en/case-studies.mdx

```diff
@@ -1,188 +1,49 @@
-
-<LinkIcon
-  iconColor="var(--colorBlack400)"
-  href="https://disinfo.quaidorsay.fr/fr/open-terms-archive/scripta-manent?service=YouTube&typeofdocument=Privacy+Policy&date1=2020-12-01&date2=2021-06-24&action=submit"
-  small={true}
-  className="mb__L"
->
-  <a target="_blank" rel="noopener">
-    See the changes
-  </a>
-</LinkIcon>
+The day before the change, the website <a target="_blank" rel="noopener" href="https://paris-luttes.info/recit-policier-de-sainte-marthe-15258?lang=fr">Paris-Luttes.info</a> ands various <a target="_blank" rel="noopener" href="https://mobile.twitter.com/OnEstLaTech/status/1434576598418796549">Internet users</a> revealed that ProtonMail had communicated the IP addresses of users linked to the Youth for Climate group to the Swiss justice system, which was itself requested by the French police <a target="_blank" rel="noopener" href="https://www.numerama.com/tech/736940-protonmail-transmet-des-adresses-ip-a-la-police-4-questions-pour-comprendre-la-polemique.html">via Europol</a>. This change mirrors what the company had indicated on <a target="_blank" rel="noopener" href="https://www.reddit.com/r/ProtonMail/comments/pil6xi/climate_activist_arrested_after_protonmail/hbqha63">Reddit</a>.
 
 ---
 
-### Google Play Store
-
-#### August 12, 2020
-
-**Google Play Store** strengthened it Google Play Store Review Guidelines on August 12, 2020. Google bans apps that falsely imply a relationship to another company/developer and apps that are improperly categorized. Using profane, vulgar, or other language that is inappropriate for a general audience in your app’s Store listing is also prohibited.
-
-<LinkIcon
-  iconColor="var(--colorBlack400)"
-  href="https://github.com/ambanum/OpenTermsArchive-versions/commit/98f6c"
-  small={true}
-  className="mb__L"
->
-  <a target="_blank" rel="noopener">
-    See the changes
-  </a>
-</LinkIcon>
+### Alibaba adds disease outbreaks to its disclaimer.
 
-#### October 9, 2021
-
-To ensure that Google Play remains a safe and respectful platform, Google now prohibits "sugar dating" apps & apps that "claim to undress people".
-
-<LinkIcon
-  iconColor="var(--colorBlack400)"
-  href="https://github.com/ambanum/OpenTermsArchive-versions/commit/f4e25e58650bc38d6f0b18b882cd5cf1d5dfdfb0"
-  small={true}
-  className="mb__L"
->
-  <a target="_blank" rel="noopener">
-    See the changes
-  </a>
-</LinkIcon>
+Covid's epidemic can be read right into Alibaba's terms of service: since May 6, 2020, the marketplace <a target="_blank" rel="noopener" href="https://github.com/ambanum/CGUs-data/commit/37503cb23">specifies</a> in cases of force majeure in which it is not responsible for order delays or failures.
```

Comment:
```suggestion
The impact of the Covid-19 pandemic can be read right into Alibaba's terms of service: since May 6, 2020, the marketplace <a target="_blank" rel="noopener" href="https://github.com/ambanum/CGUs-data/commit/37503cb23">specified</a> epidemics as cases of force majeure in which it is not responsible for order delays or failures.
```

Reply:
Done